spec/header-files.md

```diff
@@ -67,11 +88,39 @@ Each signal has its own line immediately following the record line (for single-s
 | Block size (optional) | Number of samples per block (for formats supporting block I/O). |
 | Description (optional) | Free-text description of the signal (e.g., lead name `ECG Lead II`). |
 
-**Example:**
+### Examples
+
+#### Basic header (uniform sampling frequency)(record 100 from the [MIT-BIH database](https://www.physionet.org/content/mitdb)):
+```text
```

Comment:
Similar to bulleted list. It might not make a difference for this particular renderer but it is good practice to have an empty line after a header (e.g. see: https://www.markdownguide.org/basic-syntax/ - "You should also put blank lines before and after a heading for compatibility.")